backend/wmg/data/query.py

```diff
@@ -155,7 +144,11 @@ def _query(
         attrs = [i.name for i in cube.schema if i.name in VALID_ATTRIBUTES]
         if compare_dimension is not None:
             attrs.append(compare_dimension)
-        if isinstance(criteria, FmgQueryCriteria) and compare_dimension != "dataset_id":
+        if (
```

Comment:
only add dataset_id if it's in the schema (it isn't for expression_summary_fmg now). this is for the cell counts query in marker gene calculation, since we need dataset id to calculate true population sizes per gene